src/reaktoro_pse/core/util_classes/cyipopt_solver.py

```diff
@@ -0,0 +1,35 @@
+from watertap_solvers import get_solver
+
+
+def get_cyipopt_watertap_solver(
+    max_iter=500,
+    ma27=False,
+    limited_memory=False,
+    solver_args=None,
```

Comment:
I would call this `solver_options`, because I think of `solver_args` as arguments passed in to the `solve` method.

Reply:
done